packages/create-emotion/types/tests.tsx

```diff
@@ -0,0 +1,127 @@
+import createEmotion from '../';
+
+const {
+  flush,
+  hydrate,
+  cx,
+  merge,
+  getRegisteredStyles,
+  css,
+  injectGlobal,
+  keyframes,
+  sheet,
+  caches,
+} = createEmotion({});
+
+flush();
+
+hydrate([]);
+hydrate(['123']);
+
+cx();
+cx(undefined);
+cx(null);
+cx(true);
+cx('123');
+cx('123', null, 'pf');
+cx({
+  abc: false,
+  fp: true,
+});
+cx([]);
+cx(['cl', {
+  fp: true,
+}]);
+
+merge('abc def fpfp');
+
+getRegisteredStyles([], 'abc');
+getRegisteredStyles(['abc'], 'bcd');
+getRegisteredStyles([], 'abc def fpfw');
+
+css();
+css(1);
+css('abc');
+css(true);
+
+css([]);
+css([1]);
+css([['abc', 'asdf'], 'efw']);
+
```

Comment:
Could you add a test with the string syntax, for example:
```jsx
css`
 color: ${'hotpink'};
`
```

Reply:
I added them! Thank you for your feedback!